talk/setup.tex

```diff
@@ -38,6 +38,8 @@
 % packages %
 %%%%%%%%%%%%
 
+\usepackage[utf8]{inputenc}
+\usepackage{newunicodechar}
```

Comment:
I needed those for the CI to work, but I did not need them locally. The CI is running Xelatex and I am running pdflatex.